<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aTobfile_Dialog</class>
 <widget class="QDialog" name="aTobfile_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="asmTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>301</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="binTextEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>91</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="consoleTextEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>391</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>汇编代码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>机器码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>系统输出：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>741</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_open">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_save">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_run">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_generateBin">
      <property name="text">
       <string>GenerateBin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_generateCoe">
      <property name="text">
       <string>GenerateCoe</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="useHex">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>use Hex</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
